packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -152,4 +334,19 @@ export class MigrationActions {
       throw Error(`Expected ${actual} to equal ${expected}`)
     }
   }
+
+  resetFlags () {
+    this.ctx.update((coreData) => {
+      coreData.migration.flags = {
+        hasCustomIntegrationFolder: false,
+        hasCustomIntegrationTestFiles: false,
+        hasCustomComponentFolder: false,
+        hasCustomComponentTestFiles: false,
+        hasCustomSupportFile: false,
+        hasComponentTesting: true,
+        hasE2ESpec: true,
+        hasPluginsFile: true,
+      }
```

Comment:
Should we set the default values as a constant in `packages/data-context/src/data/coreDataShape.ts` and use that one in both places? so we do not miss anything?

Reply:
sure, we can just reset using the defaults: https://github.com/cypress-io/cypress/pull/20509/commits/ec7f9ea7d9c23051efca87d5ed4efbdce7e545bb